### ISSUE TITLE
Element UI breaks fresh install

### ISSUE THREAD

**Issue by mansouryaacoubi** [nuxt/create-nuxt-app#217]
Hello,

while trying to create a fresh nuxt app, Element UI makes some problems.

### Versions
npm/npx: **6.7.0**
node: **11.12.0**

### Selected options:
custom server framework: **none**
features to install: **all**
custom ui framework: **element-ui**
custom test framework: **none**
rendering mode: **SPA**

![image](https://user-images.githubusercontent.com/6348858/55801106-9c666180-5ad5-11e9-884e-907b54ec23e2.png)

It just stuck here with installing packages..

And when I abort the package installation process and do an npm i myself, everything seems to work until I try to run `npm start` or `npm run dev`.

![image](https://user-images.githubusercontent.com/6348858/55801350-29111f80-5ad6-11e9-93ed-001642dcde42.png)

**Comment by mansouryaacoubi** [comment]
I found a solution for when the installation got stuck... I don't know why this is but it seems creating the project using the verbose flag creates the app instantly.
`npx create-nuxt-app kitchen-dashboard --verbose`

but the problem with the errors do still exist.

**Comment by mansouryaacoubi** [comment]
The problem seems to be line 27 in nuxt.config.js:

`css: ['element-ui/lib/theme-chalk/index.css'],`

![image](https://user-images.githubusercontent.com/6348858/55801704-f0be1100-5ad6-11e9-8981-d3f62e11c111.png)

**Comment by mansouryaacoubi** [comment]
@manniL I don't mind :D

**Comment by mansouryaacoubi** [comment]
Unfortunately I found out that this has to be another problem. Everything works just fine working on the same drive as the node installation. For example:

Node installation on drive C:\
Project deployment on drive C:\
=> everything works just fine

Node installation on drive C:\
Project deployment on drive D:\
=> webpack seems to have a problem with the path resolution

![image](https://user-images.githubusercontent.com/6348858/55940708-f3d51080-5c40-11e9-9912-238018831495.png)

I am not sure whether this is webpack problem or another problem.

**Comment by mansouryaacoubi** [comment]
So finally I found out, what the real problem is. Neither `create-nuxt-app` nor `webpack` are faulty.
The problem was using the Windows 10 `mklink` functionality.

I was accessing my development path through `C:\Users\Mansour\dev` which was a windows link to `D:\development`.
When I changed my directory in the `cmd` to `D:\development`, it works without any problems.
But building my project by accessing the link creates an error. I thing that maybe webpack is not yet working with windows links, since this feature is still brand new.
I will open a new issue in the webpack github.
